addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -650,7 +650,7 @@ import wUtils from '@website/js/utils';
          *      is between or !between
          * @returns {boolean}
          */
-        _compareTo(comparator, value = '', comparable, between) {
+        _compareTo(comparator, value = '', comparable, between, dependencyEl) {
```

Comment:
I prefer to avoid introducing new parameter, I don't feel we really need. I need a strong reason to keep this parameter. thanks : )

Reply:
Yes, we generally avoid introducing new parameters unless absolutely necessary. However, in this instance, passing this parameter is essential because it provides the only viable way to access information about the dependency element within the `_compareTo` method. This information is crucial for executing a valid search operation; without it, the search logic will be incomplete and will not function as intended.